libs/shared/error/src/lib/shared-error.ts

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { MultiError, VError, WError } from 'verror';
```

Comment:
Why the ESLint no-unused-vars disable?
Is it because we're importing `WError` and not using it?

Reply:
This is an initial error library that will be extended in the future. I want other engineers to know what error classes are available for future use.